doc/source/user_guide/numpy_images.rst

```diff
@@ -147,26 +152,35 @@ denote standard Cartesian coordinates, where ``x`` is the horizontal coordinate,
 ``y`` - the vertical one, and the origin is at the bottom left
 (Matplotlib axes, for example, use this convention).
 
-In the case of multichannel images, the last dimension is used for color channels
-and is denoted by ``channel`` or ``ch``.
+In the case of multichannel images, any dimension can be used for color
+channels, and is denoted by ``channel`` or ``ch``. Prior to scikit-image 0.19,
+the channels axis was always last, but in the current release the channel
+position can be specified by a ``channel_axis`` argument. Functions that
```

Comment:
In the above, is it clear to the newbie reader that dimension, axis, and position mean the same thing?

Reply:
yeah, let me rephrase this a bit and remove use of "position", at least. Hopefully the user will be able to relate "dimension" and "axis" as equivalent terms?  Maybe the first sentence here can make this clearer by reading adding a mention of axis as in:

```
... any dimension (array axis) can be used ...
```

Reply:
Great rephrasing, thanks, @grlee77!